src/chain/network_event.rs

```diff
@@ -162,23 +155,7 @@ pub struct NetworkEvent {
 impl NetworkEvent {
     /// Convert `NetworkEvent` into a Parsec Observation
     pub fn into_obs(self) -> Result<parsec::Observation<NetworkEvent, PublicId>, RoutingError> {
-        Ok(match self {
-            NetworkEvent {
-                payload: AccumulatingEvent::AddElder(id),
-                ..
-            } => parsec::Observation::Add {
-                peer_id: id,
-                related_info: Default::default(),
-            },
-            NetworkEvent {
-                payload: AccumulatingEvent::RemoveElder(id),
-                ..
-            } => parsec::Observation::Remove {
-                peer_id: id,
-                related_info: Default::default(),
-            },
-            event => parsec::Observation::OpaquePayload(event),
-        })
+        Ok(parsec::Observation::OpaquePayload(self))
```

Comment:
This doesn't have to return `Result` now.

Reply:
Looks like it didn't have to for a while already. Nice catch.